### DISCUSSION TITLE
Update to Swedish channels

### DISCUSSION THREAD

**Discussion by fredrikklevenhaus** [iptv-org/community#643] [Show and tell]
SVT1 (Geoblocked): https://svt-live-channel.akamaized.net/l1/se/svt1/master.m3u8?format=hls&defaultSubLang=1
SVT2 (Geoblocked): https://svt-live-channel.akamaized.net/l1/se/svt2/master.m3u8?format=hls&defaultSubLang=1
SVT Barn/SVT24 (Geoblocked): https://svt-live-channel.akamaized.net/l1/se/svtb/master.m3u8?format=hls&defaultSubLang=1
SVT Kunskapskanalen (Geoblocked): https://svt-live-channel.akamaized.net/l1/se/svtk/master.m3u8?format=hls&defaultSubLang=1

**Comment by ozgurkazancci** [comment]
I wish we could have SVT1 and SVT2 without geo-blocked..

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists
